R/Spectra.R

```diff
@@ -1058,14 +1082,61 @@ setReplaceMethod("isolationWindowUpperMz", "Spectra", function(object, value) {
     object
 })
 
+#' @rdname Spectra
+#'
+#' @importFrom methods getFunction
+#'
+#' @exportMethod containsMz
+setMethod("containsMz", "Spectra", function(object, mz = numeric(),
+                                            tolerance = 0,
+                                            ppm = 20, which = c("any", "all"),
+                                            BPPARAM = bpparam()) {
+    which <- match.arg(which)
+    cond_fun <- getFunction(which)
```

Comment:
Mh, not sure about `getFunction` here. What about `match.fun(match.arg(which))`?

Reply:
Fixed